pkg/kf/apps/zz_generated.clientoptions.go

```diff
@@ -242,17 +219,10 @@ func WithListFieldSelector(val map[string]string) ListOption {
 	}
 }
 
-// WithListFilters creates an Option that sets Additional filters to apply.
-func WithListFilters(val []Predicate) ListOption {
-	return func(cfg *listConfig) {
-		cfg.filters = val
-	}
-}
-
-// WithListLabelSelector creates an Option that sets A label selector.
-func WithListLabelSelector(val map[string]string) ListOption {
+// WithListFilter creates an Option that sets Filter to apply.
+func WithListFilter(val Predicate) ListOption {
```

Comment:
why change this from allowing multiple filters to just one? is having more than one filter unnecessary?

Reply:
I could only find one use of this code and it used one. K8s already does label and name based filtering server-side so I took this instance as a YAGNI feature.